pytest_reportportal/plugin.py

```diff
@@ -72,9 +72,9 @@ def pytest_sessionstart(session):
     if is_master(session.config):
         try:
             session.config.py_test_service.init_service(
-                project=config.option.rp_project
-                if session.config.getini('rp_project')
-                else session.config.option.rp_project,
+                project=session.config.option.rp_project
```

Comment:
`project=session.config.option.rp_project` is enough if you apply the suggestion I left in the pytest_configure hook.

And, BTW, the current code may be simplified the following way:
`project=session.config.option.rp_project or session.config.getini('rp_project')`

Reply:
Still not fixed. Also, please update README with information about a new option.